### DISCUSSION TITLE
Collapsible markdown inside <details><summary>...</summary></details> fails to render

### DISCUSSION THREAD

**Discussion by bittlingmayer** [community/community#22596] [Other Feature Feedback, Questions, & Ideas]
<p>I used &lt;details&gt; and &lt;summary&gt; in a very vanilla GitHub Pages-enabled repo.</p>
<p>On GitHub, the markdown in the collapsible section is rendered correctly.</p>
<p><img src="https://i.stack.imgur.com/tttYn.png" alt="enter image description here" width="533" height="181"></p>
<p>But on GitHub Pages, it is just a literal.</p>
<p><img src="https://i.stack.imgur.com/cxCLE.png" alt="enter image description here"></p>
<p>The issue is mentioned <a href="https://github.com/gettalong/kramdown/issues/155" rel="nofollow noopener"><em>Markdown inside and not being processed</em></a>, and the suggested fix is:</p>
<pre><code>markdown: kramdown
kramdown:
  parse_block_html: true
</code></pre>
<p>That does indeed fix the rendering of the content, and I am fine with switching to kramdown, but it breaks the rendering of &lt;details&gt; and &lt;summary&gt; and the twirl-down. (So I rolled it back.)</p>
<p>See the page at <a href="https://deepchar.github.io/" rel="nofollow noopener">https://deepchar.github.io/</a>, the repo at <a href="https://github.com/deepchar/deepchar.github.io" rel="nofollow noopener">https://github.com/deepchar/deepchar.github.io</a> and the StackOverflow version of this issue at <a href="https://stackoverflow.com/questions/52944720/content-of-collapsible-sections-detailssummary-renders-markdown-in-gith" rel="nofollow noopener">https://stackoverflow.com/questions/52944720/content-of-collapsible-sections-detailssummary-renders-markdown-in-gith</a>.</p>

**Comment by bittlingmayer** [comment, marked as answer]
<p>The question is now <a href="https://stackoverflow.com/questions/52944720/content-of-collapsible-sections-detailssummary-renders-markdown-in-gith/52962330#52962330" rel="nofollow noopener">answered</a>.  In short, add </p>
<pre><code>markdown: CommonMarkGhPages
</code></pre>
<p> to _config.yml, which will render the site with CommonMark instead of kramdown.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/bittlingmayer">@bittlingmayer</a>,</p>
<p>We are happy you are here! Thank you for sharing your solution. We’re looking forward to seeing you around!</p>
<p>Best,<br>
Andrea</p>

**Comment by sharegrams** [comment]
<p>This is a really cool function. I’m going to implement it somehow. Thanks for the post!</p>
